lib/galaxy/tools/__init__.py

```diff
@@ -825,6 +829,35 @@ def tests(self):
             self.__tests_populated = True
         return self.__tests
 
+    def test_data_path(self, filename):
+        repository_dir = None
+
+        # If a tool shed tool, try to find where the repository was installed. Is there a cleaner way
+        # to do this @mvdbeek?
+        if hasattr(self, 'tool_shed') and self.tool_shed:
+            repository_dir = self.tool_dir
+            while True:
+                repository_dir_name = os.path.basename(repository_dir)
+                # Eventually we should find a directory named self.changeset_revision
+                # but just in case we don't (differences between self.installed_changeset and self.changeset_revision)
+                # check for repository name also.
+                if repository_dir_name == self.changeset_revision or repository_dir_name == self.repository_name:
```

Comment:
The directory named `self.repository_name` should come first, judging by my disk layout: `~/src/shed_tools/toolshed.g2.bx.psu.edu/repos/iuc/minimap2/4070f129540a/minimap2`
If that's good enough I wouldn't bother checking for `self.changeset_revision`.

Reply:
Okay - I'll make this revision. Thanks!